Expensify/slate
# Update Slate to latest version

@NikkiWines please review

## Changes
Update Slate from origin, so that it builds with more recent ruby versions. I was going to write docs on how to update the docs and was getting a ton of ruby/bundle errors

No content change, just updating Slate sources.

Only minor changes is in source/stylesheets/screen.css.scss, because the code blocks had a missing `clear`, which caused a big vertical gap. I copied the css for those from the origin's source. They look slightly different but better imo.

## Tests

1. Run `bundle install`
1. Run `bundle exec middleman server`
1. Open the local URL that's given by middleman
1. Inspect the docs and make sure they look normal

Before | After
--- | ---
<img width="1440" alt="Screen Shot 2020-06-12 at 11 13 51 AM" src="https://user-images.githubusercontent.com/2229301/84534208-5ccd7380-ac9e-11ea-8eae-2c560186f799.png"> | <img width="1437" alt="Screen Shot 2020-06-12 at 11 13 58 AM" src="https://user-images.githubusercontent.com/2229301/84534204-59d28300-ac9e-11ea-8cfa-b58fa13f5516.png">
